desktop/ui/src/main/resources/datacleaner-home/jobs/OrderDB Customers and Employees union.analysis.xml

```diff
@@ -163,7 +166,7 @@
 												<property name="Table name" value="incomplete-records" />
 												<property name="Write mode" value="TRUNCATE" />
 												<property name="Fields"
-													value="[CONTACTFIRSTNAME,CUSTOMERNUMBER,CONTACTFIRSTNAME,CONTACTLASTNAME]" />
+													value="[JOBTITLE,CUSTOMERNUMBER,CONTACTFIRSTNAME,CONTACTLASTNAME]" />
```

Comment:
No. It was duplicate column name in staging table. I need to ask you something about the configuration of this job. 

![image](https://cloud.githubusercontent.com/assets/6339923/14169743/ed345b48-f729-11e5-9cd3-d95ea4976c1c.png)